dali/test/python/test_dali_cpu_only.py

```diff
@@ -231,6 +241,63 @@ def test_erase_cpu():
 def test_random_resized_crop_cpu():
     check_single_input(fn.random_resized_crop, size = [5, 5])
 
+def test_expand_dims_cpu():
+    check_single_input(fn.expand_dims, axes = 1, new_axis_names = "Z")
+
+def test_coord_transform_cpu():
+    check_single_input(fn.coord_transform, M = [0, 0, 1,
```

Comment:
nitpick: Maybe create a variable for M, to avoid the odd formatting.

Reply:
Done